internal/pb/cache.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package pb;
+
+option go_package = "github.com/gadget-inc/dateilager/pkg/pb";
+
+service Cache {
```

Comment:
This GRPC service is *not* what we'll use in production -- it's just for testing, but it lets us ask the daemon to do its thing in tests and in development, very convenient.

Reply:
In Prod we're going to implement a CSI proto and that's what will be called directly by K8S?

Reply:
Yup!